concent_api/utils/decorators.py

```diff
@@ -41,15 +45,22 @@ def require_golem_auth_message(view):
     @wraps(view)
     def wrapper(request, *args, **kwargs):
         if request.content_type == '':
+            log_string_message(logger, 'error: Content-Type is missing')
             return JsonResponse({'error': 'Content-Type is missing.'}, status = 400)
         elif request.content_type == 'application/octet-stream':
             try:
                 auth_message = load_without_public_key(request.body)
                 if isinstance(auth_message, message.concents.ClientAuthorization):
                     validate_golem_message_signed_with_key(auth_message, auth_message.client_public_key)
+                    log_message_received_in_endpoint(logger,
+                                                     request.resolver_match.view_name if request.resolver_match is not None else None,
+                                                     auth_message.__class__.__name__,
+                                                     auth_message.client_public_key)
```

Comment:
Someone will point that out, might as well be me. Rewrite it in this way:
```python
log_message_received_in_endpoint(
    logger,
    request.resolver_match.view_name if request.resolver_match is not None else None,
    auth_message.__class__.__name__,
    auth_message.client_public_key,
)

Reply:
done